apps/judicial-system/backend/src/app/formatters/rulingPdf.ts

```diff
@@ -64,7 +65,9 @@ function constructRestrictionRulingPdf(
     .text(
       `Þann ${formatDate(existingCase.courtStartDate, 'PPP')} heldur ${
         existingCase.judge?.name ?? '?'
-      } ${existingCase.judge?.title ?? '?'} dómþing. Fyrir er tekið mál nr. ${
+      } ${existingCase.judge?.title ?? '?'} dómþing ${lowercase(
+        existingCase.courtLocation?.replace('.', ''),
```

Comment:
What is the intent with replace here? Are you trying to remove a period at the end? I think this will remove the first period.

Reply:
Yep, you are right, this is not the correct approach. 

I've now changed this so the functionality is as follows

If the users inputs "í Héraðsdómi Reykjavíkur", we display "í Héraðsdómi Reykjavíkur."
If the users inputs "í Héraðsdómi Reykjavíkur**.**", we display "í Héraðsdómi Reykjavíkur."
If the users inputs "í Héraðsdómi Reykjavíkur**..**", we display "í Héraðsdómi Reykjavíkur.."

That is we display exactly as many periods as the user inputs and we don't touch other periods in the string, only the ones at the end. This is an acceptable solution to Gudny.